ql/pom.xml

```diff
@@ -78,8 +78,8 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>commons-configuration</groupId>
-      <artifactId>commons-configuration</artifactId>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-configuration2</artifactId>
       <version>${commons-configuration.version}</version>
```

Comment:
Shouldn't there be changes in imports as well?
like change from ``org.apache.commons.configuration.`` to org.apache.commons.configuration2.``

You should even check the maven dependency tree, if anyone else is pulling ``commons-configuration`` and exclude that & add ``org.apache.commons.configuration.`` in restricted imports